tests/EEGPSR.tex

```diff
@@ -1,129 +1,168 @@
+%	4 minutes to carry out its first command and another 4 minutes for its second command, it only has 2 minutes to carry out its third command. However, if it only took
+	2 minutes for the first and 2 minutes for the second, it still only has \eegpsrMaxCmdTime minutes for the third. To facilitate a fluent and untroubled performance of the robots, the other robots must always return or being returned to their designated position before the next robot will be given a command. If a robot moves from its designated position while another robot is working on a command, it must be immediately disabled and moved to its designated position. If a restart is still available to the team, it can be restarted at its designated position. To carry the robot, at most two team members are allowed in the arena, and the robot must be moved as quickly as possible. To start or restart the robot, at most one team member may operate the robot. The team members moving and operating the robots must leave the arena immediately after the robot is placed or started. \\
+
+	\item \textbf{Referees:} Since the score system in this test involves a subjective evaluation of the robot's behavior, the referees are EC/TC members. One referee is assigned to each team to judge performance, to measure the time for working on a command, and to keep track of the overall operating time of the robot. \\
+
+	\item \textbf{Operator:}
+	\begin{itemize}
+		\item The person operating the robot is one of the referees (default operator).
+		\item If the robot appears to consistently not be able to understand the operator, the referees ask the team to continue with a custom operator, reducing the obtained score for that command to 75\% (\refsec{asrcontinue}).
+		\item If the robot appears to consistently not be able to understand the custom operator, the referees ask the team to continue with the alternate method, reducing the obtained score for that command to 25\% (\refsec{asrcontinue}).
+	\end{itemize}
+
+	\item \textbf{Inoperative robots:} If a robot gets stuck while trying to accomplish a task during a reasonable amount of time (e.g.~30 seconds), the referee may ask the team to move back the robot to its designated position, proceeding with the next robot. \\
+
+	\item \textbf{Restarts/Changing/Charging batteries:} The team may install a charging station at the designated position of the robot, if it does not hinder the other robots. However, the robot must connect itself with the charging station after carrying out a command. Changing batteries or manually connecting the robot with the charging station is allowed during a restart. The general rules for restart apply. \\

```

Comment:
I like the idea, but only recharging during a restart is kinda harsh. On the other hand: a robot should last 30 to 40 minutes on a charge. Maybe give the teams 1 opportunity for a manual battery change taking up to max. 1-2 minutes.